### DISCUSSION TITLE
Implement different Symfony cache types

### DISCUSSION THREAD

**Discussion by ks129** [PrestaShop/PrestaShop#33413] [Ideas]
#### Is your feature request related to a problem?

Currently, PrestaShop comes with Symfony filesystem cache configuration and there is no way to select another caching way. But Symfony cache component supports many other cache types too, like Redis and Memcached.

#### Describe the solution you'd like

Implement switching between cache types and add these options to the Performance page.

#### Alternatives you've considered

Document exactly where to change code to implement another cache ways, but this solution would be hacky and would go against the rule to avoid changing PS core.

**Comment by hibatallahAouadni** [comment]
Hello @ks129 

Thank you for your suggestion. The Developer Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
